tools/js-sdk-release-tools/src/common/changelog/modifyChangelogFileAndBumpVersion.ts

```diff
@@ -3,24 +3,34 @@ import { updateUserAgent } from "../../xlc/codeUpdate/updateUserAgent.js";
 
 import fs from 'fs';
 import * as path from 'path';
-
+import { getSDKType} from "../utils.js";
 const todayDate = new Date();
 const dd = String(todayDate.getDate()).padStart(2, '0');
 const mm = String(todayDate.getMonth() + 1).padStart(2, '0'); //January is 0!
 const yyyy = todayDate.getFullYear();
 
 const date = yyyy + '-' + mm + '-' + dd;
 
-export function makeChangesForFirstRelease(packageFolderPath: string, isStableRelease: boolean) {
+export function getFirstReleaseContent(packageFolderPath: string, isStableRelease: boolean) {
```

Comment:
Remove `export` If this function is not used out of this file

Reply:
export it for ut test